wp-modules/get-version-control/tests/GetVersionControlTest.php

```diff
@@ -76,24 +76,15 @@ public function test_test_theme_contains_version_control_folder() {
 	 * Tests `get_dismissed_themes()` from the `get-version-control` module.
 	 */
 	public function test_get_dismissed_themes() {
-		$this->assertSame(
-			$this->get_mock_dismissed_themes(),
-			get_dismissed_themes(),
-		);
+		$this->assertSame( $this->get_mock_dismissed_themes(), get_dismissed_themes() );
```

Comment:
I always forget that `phpcs` doesn't do formatting like `prettier`. Strange, huh?

Reply:
Yes, very strange!